node/primitives/bridge/Cargo.toml

```diff
@@ -24,6 +24,9 @@ sp-api                 = { default-features = false, git = "https://github.com/d
 sp-core                = { default-features = false, git = "https://github.com/darwinia-network/substrate", branch = "darwinia-v0.12.1" }
 sp-runtime             = { default-features = false, git = "https://github.com/darwinia-network/substrate", branch = "darwinia-v0.12.1" }
 sp-std                 = { default-features = false, git = "https://github.com/darwinia-network/substrate", branch = "darwinia-v0.12.1" }
+# TODO: re-organize the runtime dependencies
```

Comment:
Why not do it this pr?

Reply:
I haven't come out with a good idea yet.

Reply:
Just put it under `#darwinia-network`?  Too many separations will easily lead to mess.

Reply:
Nope.
I mean runtimes depend on primitives and primitives should not depend on runtimes.

Reply:
> Just put it under `#darwinia-network`? Too many separations will easily lead to mess.

Also, in the future. There might be acala-runtime, bitforst-runtime.

Reply:
Another View: The node primitives rely on runtime primitives. That seems more reasonable.

Reply:
> Also, in the future. There might be acala-runtime, bitforst-runtime.

Add `#acala-network`, `#bitforst-network`, this way is clear enough for me.

Reply:
I definitely will consider that. But not in this PR.